### ISSUE TITLE
terminal logs scroll prematurely

### ISSUE THREAD

**Issue by rade** [weaveworks/scope#2933]
Log terminals start scrolling before reaching the full height:
![screenshot from 2017-11-12 16 49 05](https://user-images.githubusercontent.com/109109/32701245-ef0ec10c-c7c9-11e7-8d57-3ea820dd6041.png)
This also happens for popped-out log terminals. But it does _not_ happen for shell terminals. And it does _not_ happen in Weave Cloud, only standalone Scope.

I see this in both FF and Chrome.

**Comment by dlespiau** [comment]
This looks like the terminal not updating its size (and it still says 80x24 at the top).

**Comment by rade** [comment]
Right, I noticed that too. Odd that it only happens in standalone mode and only for logs.
